src/fidesops/service/pagination/pagination_strategy_link.py

```diff
@@ -50,7 +50,7 @@ def get_next_request(
         elif self.source == LinkSource.body.value:
             next_link = pydash.get(response.json(), self.path)
 
-        if next_link is None:
+        if not next_link:
```

Comment:
Was the scenario you ran into here accounted for in the tests? If not, can we add a test?

Reply:
It's accounted for end-to-end in the segment tests! For segment, next links are empty strings if they don't exist, so this was causing a bad url to be built and the traversal to fail.

Reply:
test added!